core/src/main/csharp/ch/cyberduck/core/CredentialManagerPasswordStore.cs

```diff
@@ -86,12 +86,12 @@ public override void deletePassword(Scheme scheme, int port, string hostName, st
 
         public override string findLoginPassword(Host bookmark)
         {
-            var target = ToUri(bookmark);
-            var cred = WinCredentialManager.GetCredentials(target.AbsoluteUri);
+            var cred = FindCredentials(bookmark);
             if (!string.IsNullOrWhiteSpace(cred.Password))
             {
                 return cred.Password;
             }
+
             return base.findLoginPassword(bookmark);
```

Comment:
Because the base implementation will load the platform specific implementation of the prefix service this will cause a regression in failure finding deprecated store entries.